docs/parameters.yaml

```diff
@@ -884,6 +884,13 @@ type: bool
 default: true
 components: ["director"]
 ---
+name: Director.FilteredServers
+description: >-
+  A list of server names to not to redirect client requests to.
```

Comment:
Are these names supposed to be host names or prefixes? You might also add a note that this is intended to put certain servers into downtime to give administrators a clear reason to use this config mechanism.